inductiva/molecules/scenarios/protein_solvation/post_processing.py

```diff
@@ -3,56 +3,110 @@
 import nglview as nv
 from pathlib import Path
 from typing import Literal
-from ..utils import unwrap_trajectory
+from ..utils import unwrap_trajectory, align_trajectory_to_average
+from MDAnalysis.analysis import rms
+import MDAnalysis as mda
+import pandas as pd
+from matplotlib import pyplot as plt
 
 
 class ProteinSolvationOutput:
     """Post process the simulation output of a ProteinSolvation scenario."""
 
-    def __init__(self, sim_output_path: Path = None):
+    def __init__(self,
+                 sim_output_path: Path = None,
+                 use_compressed_trajectory: bool = False):
```

Comment:
Remind that ProteinSolvationOutput will be used in the scenario.simulate(...) output. Hence, we would need to pass the argument `use_compressed_trajectory` there, which means it would be defined in the simulate.

Hence, I suggested this argument is served as an argument in every function it is used below, instead of being an attribute.